var/spack/repos/builtin/packages/likwid/package.py

```diff
@@ -87,8 +87,23 @@ def install(self, spec, prefix):
                     prefix,
                     'config.mk')
 
-        filter_file('^INSTALL_CHOWN.*',
-                    'INSTALL_CHOWN = -o $(USER)',
+# FIXME: once  https://github.com/spack/spack/issues/4432 is resolved, install as root by default and uncomment this
+#        filter_file('^INSTALL_CHOWN.*',
```

Comment:
i though one needs to keep this until installation as `root` is possible? In other words should `^INSTALL_CHOWN.*` filter be active and moved under the second `FIXME`?